docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -258,16 +268,78 @@ a socket" in the `zmq-tcp API Reference <http://api.zeromq.org/4-2:zmq-tcp>`_.
 Step 1: Configure the Network on the First Node
 -----------------------------------------------
 
-.. important::
+This step assumes an existing application development environment as described
+in :doc:`ubuntu`.
+
+#. If the first validator node is running, stop the Sawtooth components
+   (validator, REST API, and transaction processors), as described
+   in :ref:`stop-sawtooth-ubuntu-label`.
+
+#. Delete any existing blockchain data by removing all files from
+   ``/var/lib/sawtooth/``.
+
+#. (Optional) Delete existing logs by removing all files from
+   ``/var/log/sawtooth/``.
+
+#. Ensure that the required user and validator keys exist:
+
+   .. code-block:: console
+
+      $ cd; ls .sawtooth/keys/
+      yourname.priv    yourname.pub
+
+      $ ls /etc/sawtooth/keys/
+      validator.priv   validator.pub
+
+   If these key files do not exist, create them as described in
+   :ref:`generate-user-key-ubuntu`
+   and :ref:`generate-root-key-ubuntu`.
+
+#. Create a batch to initialize the Settings transaction family in the genesis
+   block.
+
+   .. code-block:: console
+
+      $ sawset genesis -k /etc/sawtooth/keys/validator.priv -o config-genesis.batch
+
+#. Create a batch to initialize the PoET consensus settings. This command sets
+   the consensus algorithm to PoET simulator, and then applies the required
+   settings.
+
+   .. code-block:: console
+
+      $ sawset proposal create -k /etc/sawtooth/keys/validator.priv \
+      -o config.batch
+      sawtooth.consensus.algorithm=poet \
+      sawtooth.poet.report_public_key_pem={simulator_rk_pub.pem} \
```

Comment:
This should be:  `sawtooth.poet.report_public_key_pem=$(cat /etc/sawtooth/simulator_rk_pub.pem)`